Cesium.externs.js

```diff
@@ -2585,6 +2585,56 @@ Cesium.DataSource = function() {};
 /**
  * @constructor
  */
+Cesium.GeoJsonDataSource = function() {};
+
+/**
+ * @typedef {{
+ *   source: string,
+ *   describe: Function,
+ *   markerSize: number,
+ *   markerSymbol: string,
+ *   markerColor: Cesium.Color,
+ *   stroke: Cesium.Color,
+ *   strokeWidth: number,
+ *   fill: Cesium.Color,
+ *   clampToGround: boolean
+ * }}
+ */
+Cesium.GeoJsonDataSourceLoadOptions;
+
+/**
+ * @param {string|Object} data
+ * @param {!Cesium.GeoJsonDataSourceLoadOptions} options
+ * @returns {Cesium.DataSource|Promise.<Cesium.DataSource>}
```

Comment:
There should be no s in `@return`. Could you also fix the other occurences in the file?